Exercise_2/index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css"/>
+</head>
+<body>
+ <header class="header">
+        <aside class="header__side-section">Social</aside>
+        <section class="header__central-section header__central-section_focus">City Guide</section>
+        <aside class="header__side-section">Account</aside>
+    </header>
+    <nav class="navigation">Main nav</nav>
+    <main class="main">
+        <aside class="main__side-nav">SideNav</aside>
+        <section class="main__main">Main</section>
```

Comment:
Please, avoid using class names like `main__main`. Probably it is better to use `main__content`. Right now it doesn't matter as we don't have a big structure, but in the next task, you will have to think about class names.